### DISCUSSION TITLE
form markup

### DISCUSSION THREAD

**Discussion by marcelmoreau** [imarc/boilerplate#39] [General]
Where is a good area in which we can put in form markup, see it in the browser, etc? It's likely not something part of the atomic design layout. Seems like it might be more baseline content?

@khamer

**Comment by khamer** [comment]
So more form controls (inputs, textareas, etc) would just be atoms, similar to buttons and toggle buttons. they should within regular atomic design. If we need to style controls in a embedded form we don't control, I'd handle that the same way we'd handle embedded text, and do something like

```scss
.embeddedForm {
  button {
    @extend .button;
  }
  // ...
}
```

But I think we want to avoid styling naked html elements like `<input>` tags especially, since they get incorporated into other components.
